integration-tests/kafka-connect-avro-configs/file-sink-standalone.properties

```diff
@@ -13,4 +13,4 @@ internal.value.converter.schemas.enable=true
 # with the converter we want to apply it to
 value.converter.endpoint=https://glue-gamma.us-east-1.amazonaws.com
 value.converter.region=us-east-1
-value.converter.avroRecordType=GENERIC_RECORD
\ No newline at end of file
+value.converter.avro.record.type=GENERIC_RECORD
```

Comment:
Does this break configs of existing users?

Reply:
The Constant names are not changing. There is a risk of customers are using it with the name directly rather than using constant names. But this will be a version bump, so existing customers will not break.